socorro/external/es/base.py

```diff
@@ -4,6 +4,88 @@
 
 import datetime
 
+from configman import Namespace, class_converter
+
+from socorro.app.generic_app import App
+
+
+def string_to_list(input_str):
+    return [x.strip() for x in input_str.split(',') if x.strip()]
```

Comment:
configman has a ``str_to_list`` converter, and it's associated with the ``list`` type, so I think this can be avoided entirely. See https://github.com/mozilla/configman/blob/master/configman/converters.py#L300 and https://github.com/mozilla/configman/blob/master/configman/converters.py#L336